spec/models/post_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:author) { User.create(name: 'Test User', bio: 'This is my bio', postsCounter: 0) }
+
+  describe 'validations' do
+    context 'when title is blank' do
+      it 'is invalid' do
+        post = Post.new(title: '', text: 'Test post text', author: author, comments_counter: 0, likes_counter: 0)
+        expect(post).to be_invalid
+        expect(post.errors[:title]).to include("can't be blank")
+      end
+    end
+
+    context 'when title is too long' do
+      it 'is invalid' do
+        post = Post.new(title: 'a' * 251, text: 'Test post text', author: author, comments_counter: 0, likes_counter: 0)
+        expect(post).to be_invalid
+        expect(post.errors[:title]).to include('is too long (maximum is 250 characters)')
+      end
+    end
+
+    context 'when comments_counter is negative' do
+      it 'is invalid' do
+        post = Post.new(title: 'Test Post', text: 'Test post text', author: author, comments_counter: -1,
+                        likes_counter: 0)
+        expect(post).to be_invalid
+        expect(post.errors[:comments_counter]).to include('must be greater than or equal to 0')
+      end
+    end
+
+    context 'when likes_counter is negative' do
+      it 'is invalid' do
+        post = Post.new(title: 'Test Post', text: 'Test post text', author: author, comments_counter: 0,
+                        likes_counter: -1)
+        expect(post).to be_invalid
+        expect(post.errors[:likes_counter]).to include('must be greater than or equal to 0')
+      end
+    end
+  end
```

Comment:
- [ ] Kindly add tests for the `most_recent_comments` method in your Post model.

- [ ] As requested by the initial reviewer, kindly add more key features in your readme doc.

   ![image](https://user-images.githubusercontent.com/54848363/230638857-2b7d3807-ec10-4399-b195-15e276f72b79.png)